CMakeLists.txt

```diff
@@ -75,6 +75,10 @@ TRIBITS_ADD_OPTION_AND_DEFINE(
     LIST(APPEND DEVICE_LIST "<Cuda,CudaUVMSpace>")
   ENDIF()
 
+  IF( Trilinos_ENABLE_COMPLEX_DOUBLE AND ((NOT DEFINED CMAKE_CXX_USE_RESPONSE_FILE_FOR_OBJECTS) OR (NOT CMAKE_CXX_USE_RESPONSE_FILE_FOR_OBJECTS)) )
```

Comment:
This is fine, though it may be possible to turn OFF `KokkosKernels_INST_COMPLEX_DOUBLE` even if `Trilinos_ENABLE_COMPLEX_DOUBLE` is ON.

Reply:
@mhoemmen Thanks for the catch! I'm guessing most users use `Trilinos_ENABLE_COMPLEX_DOUBLE` so I think I should leave that and add a check for  KokkosKernels_INST_COMPLEX_DOUBLE also, does this sound correct?

Reply:
@ndellingwood What about stand-alone builds?  Does `Trilinos_ENABLE_COMPLEX_DOUBLE` even exist in that case?

Reply:
@mhoemmen I'm not sure that kokkos-kernels has standalone CMake support, I think for standalone builds the Makefile system should be used but I may be wrong.